index.html

```diff
@@ -0,0 +1,144 @@
+            </ul>
+            </nav>
+        </div>
+        <div class="main-content">
+        <i id="hamburger" class="fa-solid fa-bars fa-2x"></i>
+        <h3 class="hello-concert">"Hello! Welcome Gen Mex"</h1>
+        <h1 class="concert-title"><b>GEN MEX CONCERT<br>AMAZON ORIGINALS<br> 2023</b></h1>
+        <p class="concert-paragraph"> A joyful event at the NeueHouse Hollywood.
+        The event brings together a new generation of mexican fresh stars who will delight 
+        the audience with some of their best hits.
+        </p>
+        <p class="date-and-time">2023.03.03(FRI) ~ 05(SUN)</p>
+        <p class="place"><b>@NeueHouse Hollywood, Columbia Square, USA</b></p>
+        </div>
+    </header>
+    <main class="main-program">
+        <h3 class="main-program-title">Main Program</h3>
+        <hr class="breakline">
+        <div class="main-container-cards">
+        <div class="main-container">
+            <div class="main-container-icon">
+                <i class="fa-solid fa-microphone fa-5x"></i>               
+            </div>
+            <h3 class="title-main-container">Opening</h3>
+            <p class="description-main-container">The youngest singers from Mexico will start this night with their awesome talent</p>
+        </div>
+        <div class="main-container">
+            <div class="main-container-icon">
+                <i class="fa-solid fa-user fa-5x"></i>            
+            </div>
+            <h3 class="title-main-container">Main Artists</h3>
+            <p class="description-main-container">Three out of our main artists for today will make us enjoy for 2 hours of their most popular songs</p>
+        </div>
+        <div class="main-container">
+            <div class="main-container-icon">
+                <i class="fa-solid fa-bacon fa-5x"></i>
+            </div>
+            <h3 class="title-main-container">Break</h3>
+            <p class="description-main-container">In this time you will enjoy with your friends while listening to diverse music, courtesy of the DJ. Let's enjoy</p>
+        </div>
+        <div class="main-container">
+            <div class="main-container-icon">
+                <i class="fa-solid fa-microphone fa-5x"></i>               
+            </div>
+            <h3 class="title-main-container">Opening</h3>
+            <p class="description-main-container">Another group of the youngest singers from Mexico will re-start this night with their awesome talent</p>
+        </div>
+        <div class="main-container">
+            <div class="main-container-icon">
+                <i class="fa-solid fa-user fa-5x"></i>            
+            </div>
+            <h3 class="title-main-container">Closing</h3>
+            <p class="description-main-container">The last main artists for today will make us enjoy for 2 hours of their most popular songs. Let's close this night!!</p>
+        </div>
+        </div>
```

Comment:
- [ ] While you did a good job staying each card inside your 'Main program' section :+1:. As mentioned before you should align all the elements to the middle of the card to match the Figma template as you can see below.

- [OPTIONAL] Kindly check the background of each one of the cards to match with the Figma :+1: 

**Your design**

![Screenshot from 2023-02-16 08-38-41](https://user-images.githubusercontent.com/105117832/219380989-533d8c91-f4f5-4907-a124-8b3c14c372d4.png)

**Figma**

![Screenshot from 2023-02-16 08-44-10](https://user-images.githubusercontent.com/105117832/219381220-72c4bde1-a2c5-426a-a744-e6c9ae656b79.png)

Reply:
done